patches/media-media_options.gni.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/media/media_options.gni b/media/media_options.gni
+index 0c62e9b40706d4b03cec9f00d976d4061b336b50..01561cf6ba297102fc334386f040f578b6fbbfff 100644
+--- a/media/media_options.gni
++++ b/media/media_options.gni
+@@ -9,6 +9,7 @@ import("//build/config/jumbo.gni")
+ import("//media/gpu/args.gni")
+ import("//testing/libfuzzer/fuzzer_test.gni")
+ import("//third_party/libaom/options.gni")
++if (brave_chromium_build) { import("//brave/build/config.gni") }
+ 
+ # Do not expand this list without double-checking with OWNERS, this is a list of
+ # all targets which roll up into the //media component. It controls visibility
+@@ -164,6 +165,9 @@ declare_args() {
+   # Windows and Mac.
+   enable_cdm_host_verification =
+       enable_library_cdms && (is_mac || is_win) && is_chrome_branded
++  if (brave_chromium_build) {
```

Comment:
why are we patching for something inside declare_args? Just set `enable_cdm_host_verification` in config.js
This patch should be removed

Reply:
Oops. you're right. I thought this is outside of `declare_args() block`

Reply:
But it also checks `enable_library_cdms`